Sources/Widgets/Core/WidgetManager/api.md

```diff
@@ -31,3 +34,9 @@ Grabs the focus of the view widgets associated with the widget factory.
 
 Clears focus flag for any focused widgets.
 
+## setCaptureOn(WidgetManager.CaptureOn)
+
+Sets the picking buffer capture mode. There are two possible values:
+`CaptureOn.MOUSE_RELEASE` and `CaptureOn.MOUSE_MOVE`.
+- `MOUSE_RELEASE` will capture the picking buffer whenever any mouse button is released. This will capture the entire visible rendering surface and make it available for picking. This means capture phase may take longer for larger screens, but picking will be lightning-fast for static scenes.
```

Comment:
Maybe add `Picking buffer will not be refreshed if widget state is modified programatically without mouse button being released (e.g. widget modified from the GUI)`